src/toggle-switch/toggle-switch.scss

```diff
@@ -168,7 +167,30 @@
     // Focus on non-disabled toggle switch
     &:enabled:focus ~ .iui-toggle {
       @include themed {
-        box-shadow: t(iui-focus-box-shadow);
+        outline: 2px solid t(iui-color-foreground-primary);
+      }
+      outline-offset: -1px;
+    }
+
+    &:enabled:checked:focus ~ .iui-toggle::after {
+      content: '';
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      top: 0;
+      left: 0;
+      border-radius: $iui-baseline;
+      box-sizing: border-box;
+      @include themed {
+        border: 1px solid t(iui-color-foreground-accessory);
```

Comment:
I reworked the focus on the toggle switches to look a little better.  This could be an issue if something is right up against the toggle switch where the offset outline might clash with it, but that _shouldn't_ happen.

![Screen Shot 2022-01-05 at 1 55 56 PM](https://user-images.githubusercontent.com/849817/148273068-fd3f714a-2a15-44a4-80c5-8f0c82002253.png)
![Screen Shot 2022-01-05 at 1 55 47 PM](https://user-images.githubusercontent.com/849817/148273070-f7d8281c-190b-4cc5-a684-4f8fbd729863.png)